src/Symfony/Component/Notifier/Bridge/Slack/Block/SlackButtonBlockElement.php

```diff
@@ -16,20 +16,27 @@
  */
 final class SlackButtonBlockElement extends AbstractSlackBlockElement
 {
-    public function __construct(string $text, string $url, ?string $style = null)
+    public function __construct(string $text, ?string $url = null, ?string $style = null, ?string $value = null)
```

Comment:
I'm not sure this change makes sense. Buttons defined this way are legacy anyway?
What would the value be used for, for buttons?